server/service/src/sync/program_schemas/hiv_testing_encounter_ui_schema.json

```diff
@@ -329,11 +329,13 @@
             ]
           }
         },
+        { "type": "Spacer", "options": { "margin": 10 } },
```

Comment:
This group only has one element, so looks a bit nicer with a smaller margin. (And a good case for having the option ;) )